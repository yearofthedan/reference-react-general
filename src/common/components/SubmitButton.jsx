import React from "react";

const SubmitButton = ({children}) => (
    <button type="submit">{children}</button>
);

export default SubmitButton;
