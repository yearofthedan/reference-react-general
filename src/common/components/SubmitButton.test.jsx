import React from "react";
import SubmitButton from "./SubmitButton";
import renderer from "react-test-renderer";

describe('<SubmitButton />', () => {
   it('renders to its snapshot', () => {
       const tree = renderer
           .create(<SubmitButton />)
           .toJSON();
       expect(tree).toMatchSnapshot();
   });
});
