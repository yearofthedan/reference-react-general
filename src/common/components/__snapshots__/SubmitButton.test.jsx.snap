// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitButton /> renders to its snapshot 1`] = `
<button
  type="submit"
/>
`;
